staging/vhost-device-gpu/CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 ### Added
 
 ### Changed
+- Rename --gpu-mode virgl-renderer to "virglrenderer"
```

Comment:
What about renaming also the feature (now called `virgl_renderer`?

Reply:
BTW, we didn't release any version, so I don't think we need an entry in the changelog